### PR TITLE
Replace build tool from next.js to vite

### PR DESCRIPTION
**Pull request by WatanabeToshimitsu**
## What?
Replace build tool from next.js to vite

## Why?
https://vitejs.dev/guide/why.html

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/217#" title="Author self-approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [WatanabeToshimitsu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by WatanabeToshimitsu** [comment]
@d-hayashi 
変更量が大きすぎて見るのは厳しいと思うので，動かして問題なければ通してもらえればと思います！
もし新しいリポジトリ作る方が良さそうならそうします

File: package.json
Comment (d-hayashi) [inline package.json:13]: 修正お願いします

File: package.json
Comment (d-hayashi) [inline package.json:2]: 修正お願いします

File: README.md
Comment (d-hayashi) [inline README.md:18]: vite でも `NEXT_` のプレフィックスを使う感じです？

**Review by d-hayashi — COMMENTED** [review]
PRあざます！
コメントのチェックをお願いします

File: README.md
Comment (WatanabeToshimitsu) [reply README.md]: README の変更忘れてました！
[任意のプレフィクスで環境変数を process.env に取り込めるプラグイン](https://github.com/IndexXuan/vite-plugin-env-compatible)を導入して，DATAWARE_TOOLS_ をプレフィクスにしています！

**Comment by WatanabeToshimitsu** [comment]
@d-hayashi リマインド!!

File: README.md
Comment (d-hayashi) [reply README.md]: なるほど！

**Review by d-hayashi — APPROVED** [review]
LGTM!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/217#pullrequestreview-818502739" title="Approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/217#" title="Author self-approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [WatanabeToshimitsu,d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
